plugins/ltac/g_auto.mlg

```diff
@@ -249,7 +249,8 @@ END
 VERNAC COMMAND EXTEND HintCut CLASSIFIED AS SIDEFF
 | #[ locality = Attributes.locality; ] [ "Hint" "Cut" "[" hints_path(p) "]" opthints(dbnames) ] -> {
         let entry = Hints.HintsCutEntry (Hints.glob_hints_path p) in
-        Hints.add_hints ~local:(Locality.make_section_locality locality)
+        let locality = if Locality.make_section_locality locality then Goptions.OptLocal else Goptions.OptGlobal in
+        Hints.add_hints ~locality
```

Comment:
Is it normal that `Hint Cut` does not accept the new `export` attribute?

Reply:
The whole hint registering system is bonkers. It needs a complete rewriting with a preliminary serious design phase for the semantics of the commands. This PR was born of the urgency of covering 95% of the modularity-breaking code, and even though I think this one can be handled as hackishly as we are doing it for standard hints, I am not too confident it will behave that well...

Reply:
Since the affected commands are `Hint Resolve`, `Hint Immediate`, `Hint Unfold` and `Hint Extern`, it would be better to clearly document that these are the four commands that accept the `export` attribute and avoid a new "documentation is lying" bug report. I guess extending the support to `Hint Mode`, `Hint Cut`, `Hint Transparent` and `Hint Opaque` is up to @mattam82.

Reply:
Done.

Reply:
🎉! Wouldn't instances fall in the 95%? But I'm happy with an issue :-)